binaries/build.py

```diff
@@ -0,0 +1,80 @@
+import argparse
+import glob
+import os
+import sys
+
+# To help discover local modules
+REPO_ROOT = os.path.join(os.path.dirname(os.path.abspath(__file__)), "..")
+sys.path.append(REPO_ROOT)
+
+
+def build_dist_whl(args):
+    """
+    Function to build the wheel files for torchserve, model-archiver and workflow-archiver
+    """
+    if args.nightly:
+        print(
+            "## Started pippy build"
+        )
+        create_wheel_cmd = "python setup.py "
+    else:
+        print("## Started pippy build")
+        create_wheel_cmd = "python setup.py bdist_wheel "
```

Comment:
is "--release" needed at here?
create_wheel_cmd = "python setup.py bdist_wheel " =>
create_wheel_cmd = "python setup.py bdist_wheel --release"